LethalAvatars/PlayerAvatarAPI.cs

```diff
@@ -161,7 +161,8 @@ public static void ApplyNewAvatar(Avatar clonedAvatar, PlayerControllerB player,
         avatarDriver.lastServerItemGrab = player.serverItemHolder;
         Transform cameraTransform = metarig.Find("CameraContainer/MainCamera");
         avatarDriver.AnimationDone(clonedAvatar.GetComponent<Animator>(), cameraTransform, player.IsLocal());
-        registeredAvatars.Add(player, clonedAvatar);
+        avatarDriver.SetupAvatar();
```

Comment:
`SetupAvatar()` should occur *before* `InitializeAnimatorControllers(Avatar)` is invoked, because this will ensure that no animators interfere with `SetupAvatar()` process.